src/containers/chat/account_screen/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Route } from 'react-router-dom'
+import Login from '../../login'
+import Register from '../../register'
+
+import './index.css'
+
+const Account_Screen = () => (
+<div class="chatbox">
+    <div className="incoming">
```

Comment:
Give html and css classes proper names. Giving a div a class name of "msgBox1" for example does not make sense. What does the 1 mean. How does "msgBox1" differ from "msgBox2". Why are they styled differently?